src/django/api/views/v1/opensearch_query_builder/opensearch_query_builder.py

```
@@ -96,12 +96,46 @@ def add_geo_distance(self, field, lat, lng, distance):
-        pass
+    def add_search_after(self, search_after_value, search_after_id, id_type):
+        '''
+        We need to get current order (asc/desc)
+        from already built OpenSearch query.
+        '''
+        for sort_item in self.query_body.get('sort', []):
+            for key, value in sort_item.items():
+                if isinstance(value, dict) and 'order' in value:
+                    currentSorting = value['order']
+                elif key == 'order':
+                    currentSorting = value['order']
```

Ruff (0.7.0)

Combine `if` branches using logical `or` operator

Combine `if` branches

(SIM114)